src/input_utils.py

```diff
@@ -17,13 +17,15 @@
 import attr
 import tensorflow as tf
 import tensorflow_text as tf_text
+from einops import reduce, rearrange
```

Comment:
Nit: imports should be for modules or packages, not individual functions.  https://google.github.io/styleguide/pyguide.html#22-imports